TAG-S&P-questionnaire.md

```diff
@@ -0,0 +1,106 @@
+For more information for this type of attack, see this section of the explainer.
+
+### 2.6. What information from the underlying platform, e.g. configuration data, is exposed by this specification to an origin?
+
+This feature should not reveal any additional information about the user's underlying platform.
+
+### 2.7. Does this specification allow an origin access to sensors on a user’s device
+
+No.
+
+### 2.8. What data does thisi UI specification expose to an origin? Please also document what data is identical to data exposed by other features, in the same or different contexts.
+
+Partitioned cookies do not expose any additional information to an origin that unpartitioned cross-site cookies do not already.
+Instead, that same data is now scoped to the top-level site instead of being available across different top-level contexts.
+
+### 2.9. Does this specification enable new script execution/loading mechanisms?
+
+No.
+
+### 2.10. Does this specification allow an origin to access other devices?
+
+No.
+
+### 2.11. Does this specification allow an origin some measure of control over a user agent’s native UI?
+
+No.
+
+### 2.12. What temporary identifiers might this specification create or expose to the web?
+
+This proposal is a change to an existing identifier, cookies.
+The purpose of this design is to give third-party contexts the ability to use these identifiers for activity scoped within a single top-level site that is not a vector for cross-site tracking.
+
+### 2.13. How does this specification distinguish between behavior in first-party and third-party contexts?
+
+The Partitioned attribute is designed for use in third-party contexts, but that does not mean that they cannot be used in first-party contexts.
+When used in first-party contexts, their semantics are identical to that of first-party cookies.
+
+### 2.14. How does this specification work in the context of a user agent’s Private \ Browsing or "incognito" mode?
+
+Like all other cookies in private browsers, partitioned cookies should be cleared when a private browsing session ends.
+Some browser private settings also block the use of cross-site cookies.
+User agents may wish to implement exceptions for partitioned cookies for these cross-site cookie restrictions for private browsing mode.
+
+### 2.15. Does this specification have a "Security Considerations" and "Privacy Considerations" section?
+
+[Yes](https://github.com/WICG/CHIPS/blob/main/README.md#security-and-privacy-considerations).
+
+### 2.16. Does this specification allow downgrading default security characteristics?
+
+No.
+
+## 3. Threat Models
+
+### 3.1. Passive Network Attackers
+
+The Partitioned attribute can only be used for cookies being sent over secure connections, so the cookie strings are not visible to passive network attackers.
+
+### 3.2. Active Network Attackers
+
+See 3.1.
+
+### 3.3. Same-Origin Policy Violations
+
+None.
```

Comment:
`__Host` cookies still violate the same-origin policy by allowing different ports to access the same cookie, right?

Reply:
Good point. Changed.